OpenRA.Mods.Common/Traits/TargetableUnit.cs

```diff
@@ -9,28 +9,33 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Linq;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Actor can be targeted.")]
-	public class TargetableUnitInfo : ITraitInfo, ITargetableInfo
+	public class TargetableUnitInfo : UpgradableTraitInfo, ITraitInfo, ITargetableInfo
```

Comment:
This conflicts with https://github.com/OpenRA/OpenRA/pull/8662 and somehow broke things. The defending allied units in the shellmap don't retaliate anymore.